javascriptlib/src/mill/javascriptlib/TypeScriptModule.scala

```diff
@@ -89,8 +179,8 @@ trait TypeScriptModule extends Module { outer =>
       )
     )
 
-    os.call(npmInstall().path / "node_modules/.bin/tsc")
-    os.copy.over(millSourcePath, Task.dest / "typescript")
+    os.copy(millSourcePath, Task.dest / "typescript", mergeFolders = true)
```

Comment:
What is the purpose of this `os.copy`?

Reply:
moving source files, running with ts-node in development